src/main/java/hudson/plugins/accurev/AccurevSCM.java

```diff
@@ -124,6 +125,19 @@ public AccurevSCM(String serverName, String depot, String stream) {
     updateMode();
   }
 
+  @DataBoundConstructor
+  public AccurevSCM(String serverName, String serverUUID, String depot, String stream) {
```

Comment:
Why is this change needed? if so should the deprecated method not call this one.

Reply:
One of the tests needed this constructor - i'll simplify the deprecated one to call this